### ISSUE TITLE
_splpy_to_tuples() missing 2 required positional arguments: 'fn' and 'attributes'.

### ISSUE THREAD

**Issue by ddebrunner** [IBMStreams/streamsx.topology#736]
Invalid Python code (e.gf.  simple typo) in a SPL Python operators leads to a confusing error.
```
from streamsx.spl import spl

def splNamespace():
    return "ab"

@spl.map()
class S(object):
    def __init__(self, a):
        self.a = a
        self.b = false

    def __call__(self, *tuple):
        return tuple

```

```
09 Feb 2017 14:24:52.681 [16166] ERROR #splapptrc,J[0],P[0],B,python M[splpy_general.h:callFunction:242]  - Failed function execution streamsx.spl.runtime._splpy_to_tuples
09 Feb 2017 14:24:52.684 [16166] ERROR #splapplog,J[0],P[0],B,spl_pe M[PEImpl.cpp:logTerminatingException:1893]  - CDISR5033E: An exception occurred during the execution of the B operator. Processing element number 0 is terminating.
09 Feb 2017 14:24:52.685 [16166] ERROR #splapptrc,J[0],P[0],B,spl_operator M[PEImpl.cpp:instantiateOperators:615]  - CDISR5030E: An exception occurred during the execution of the B operator. The exception is: _splpy_to_tuples() missing 2 required positional arguments: 'fn' and 'attributes'
09 Feb 2017 14:24:52.685 [16166] ERROR #splapptrc,J[0],P[0],B,spl_pe M[PEImpl.cpp:process:1270]  - CDISR5079E: An exception occurred during the processing of the processing element. The error is: _splpy_to_tuples() missing 2 required positional arguments: 'fn' and 'attributes'.
09 Feb 2017 14:24:52.707 [16166] ERROR #splapptrc,J[0],P[0],B,spl_operator M[PEImpl.cpp:process:1301]  - CDISR5053E: Runtime failures occurred in the following operators: B.
```

**Comment by ddebrunner** [comment]
With a fix the output is:
```
[streamsadmin@streamsqse pyops]$ output/bin/standalone
09 Feb 2017 16:21:38.939 [14938] ERROR #splapptrc,J[0],P[0],B,python M[B.cpp:B$OP:68]  - Fatal error: cannot create instance of class S736 in module s
Traceback (most recent call last):
  File "/tmp/8580372447746845834/output/toolkits/pyops/opt/.__splpy/packages/streamsx/spl/spl.py", line 62, in __init__
    self.__splpy_instance = wrapped(*args,**kwargs)
  File "/tmp/8580372447746845834/output/toolkits/pyops/opt/python/streams/s.py", line 10, in __init__
    self.b = false
NameError: name 'false' is not defined
09 Feb 2017 16:21:38.944 [14938] ERROR #splapplog,J[0],P[0],B,spl_pe M[PEImpl.cpp:logTerminatingException:1893]  - CDISR5033E: An exception occurred during the execution of the B operator. Processing element number 0 is terminating.
09 Feb 2017 16:21:38.944 [14938] ERROR #splapptrc,J[0],P[0],B,spl_operator M[PEImpl.cpp:instantiateOperators:615]  - CDISR5030E: An exception occurred during the execution of the B operator. The exception is: name 'false' is not defined
09 Feb 2017 16:21:38.944 [14938] ERROR #splapptrc,J[0],P[0],B,spl_pe M[PEImpl.cpp:process:1270]  - CDISR5079E: An exception occurred during the processing of the processing element. The error is: name 'false' is not defined.
09 Feb 2017 16:21:38.967 [14938] ERROR #splapptrc,J[0],P[0],B,spl_operator M[PEImpl.cpp:process:1301]  - CDISR5053E: Runtime failures occurred in the following operators: B.

```

**Comment by ddebrunner** [comment]
Fixed with pr #737
